XenModel/FriendlyNames.resx

```diff
@@ -1076,6 +1076,15 @@
   <data name="Message.body-vcpu_qos_failed" xml:space="preserve">
     <value>Quality of Service settings for VCPU '{0}' on virtual machine '{1}' could not be obeyed.</value>
   </data>
+  <data name="Message.body-vdi_cbt_metadata_inconsistent" xml:space="preserve">

```

Comment:
We need better messages here, both for the body and for the name.

Reply:
Below are my suggestions for the name and body of the messages. Note that I have used 2 different sets of instructions in the body (italicized) as I wasn't sure if the first was clear enough. Please do not take this as definitive so feel free to change them if you think the wording is dodgy or unclear.

**Inconsistent**
Name: Changed block tracking disabled due to inconsistent metadata.
Body: Changed block tracking disabled for VDI {0} due to inconsistent metadata. _To continue using changed block tracking with VDI {0} re-enable changed block tracking, take a snapshot and back it up._

**Resize**
Name: Changed block tracking disabled due to VDI resizing failure.
Body: Changed block tracking disabled for VDI {0} due to VDI resizing failed. _To continue using changed block tracking with VDI {0} re-enable changed block tracking, take a snapshot and back it up._

**Snapshot**
Name: Changed block tracking disabled due to snapshot failure.
Body: Changed block tracking disabled for VDI {0} due to snapshot failure. _To continue using changed block tracking with VDI {0} you will need to re-do the whole backup setup including re-enabling changed block tracking, taking a snapshot and backing it up._